lib/earmark_parser/helpers/ast_helpers.ex

```diff
@@ -28,15 +28,16 @@ defmodule Earmark.Parser.Helpers.AstHelpers do
   end
 
   @doc false
-  def augment_tag_with_ial(tags, ial)
-
-  def augment_tag_with_ial([{t, a, c, m} | tags], atts) do
-    [{t, merge_attrs(a, atts), c, m} | tags]
+  def augment_tag_with_ial(tags, ial, src)
+  def augment_tag_with_ial([{t, a, c, m}|tags], atts, _src) do
+    [{t, merge_attrs(a, atts), c, m}|tags]
   end
-
-  def augment_tag_with_ial([], _atts) do
+  def augment_tag_with_ial([], _atts, _src) do
     []
   end
+  def augment_tag_with_ial([any], _atts, src) do
+    [any <> src]
```

Comment:
and here I use the fallback, we do not have an element, therefore we have escaped text and just append the unparsed IAL string again